packages/dashboard/stories/MnemonicConfirm.tsx

```diff
@@ -0,0 +1,11 @@
+import { story } from '@dimensiondev/maskbook-storybook-shared'
+import { MnemonicConfirm as C } from '../src/components/Mnemonic/MnemonicConfirm'
+
+const { meta, of } = story(C)
+export default meta({
+    title: 'Components/Mnemonic Confirm',
+    argTypes: { onChange: { action: 'change' } },
```

Comment:
suggest to use @storybook/actions instead of argTypes. Provide it in args bellow. it's more typed